modules/statics/src/coins.ts

```diff
@@ -1382,4 +1383,5 @@ export const coins = CoinMap.fromCoins([
     AccountCoin.DEFAULT_FEATURES
   ),
   tfiatToken('tfiat:USD', 'FIAT US Dollar', UnderlyingAsset.USD, 2),
+  fiatToken('fiat:USD', 'FIAT US Dollar', UnderlyingAsset.USD, 2, Networks.main.fiat),
```

Comment:
Should network not be implied for `fiatToken`, since we differentiate between then with a `tfiatToken` and `fiatToken`? Would expect a default value here. Looking at all the other definitions, there is only one case where a non-default network is used, and thats georili ETH, which makes sense. We wont have a Fiat thats not on the fiat network. And if we do, it will be an exception, not the standard.

Reply:
Ok, but as it's not defined into the fiatToken method ; Which should be the default value in this case ? maybe @emiacc  can help here too.

Reply:
I would expect `Networks.main.fiat` would be the default network for fiat tokens